src/domain/application/applicationView/ApplicationView.tsx

```diff
@@ -138,17 +145,20 @@ function ApplicationView({ application, hanke, signedInUser, onEditApplication }
           </SectionItemContent>
           <SectionItemTitle>{t('hakemus:labels:applicationState')}:</SectionItemTitle>
           <SectionItemContent>
-            <Box display="flex">
-              <Box as="span" mr="var(--spacing-2-xs)">
+            <Box>
+              <Box mb="var(--spacing-2-xs)">
                 <ApplicationStatusTag status={alluStatus} />
               </Box>
-              {alluStatus === AlluStatus.DECISION && (
-                <DecisionLink
-                  applicationId={id}
-                  linkText={t('hakemus:labels:downloadDecisionShort')}
-                  filename={applicationIdentifier}
-                />
-              )}
+              {alluStatus === AlluStatus.DECISION &&
```

Comment:
Decision(s) are not displayed if AlluStatus is `OPERATIONAL_CONDITION ` or `FINISHED`, even though they should be? Same is in `ApplicationListItem.tsx`.

Reply:
Good call! I completely missed this and just followed the existing logic of status being DECISION. Fixed now. Though now I wonder whether the decision links should be shown always if there are (current) decisions.

Reply:
The status shouldn't matter. If there are decisions, we can show them.

There are several other statuses between DECISION, OPERATIONAL CONDITION and FINISHED as well.

Reply:
I changed it to show the decision links if there are decisions no matter what the status is.